packages/site/src/data/graphql/libraries.ts

```diff
@@ -1,26 +1,490 @@
+		author: "Meta",
+		gitHubRepo: "https://github.com/graphql/graphql-js",
+		npmPackage: "graphql",
+		href: "https://graphql.org/graphql-js/",
+		description:
+			"The JavaScript reference implementation for GraphQL, a query language for APIs created by Facebook.",
+		image: "https://github.com/graphql.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Mercurius",
+		author: "Pablo Sáez",
+		gitHubRepo: "https://github.com/mercurius-js/mercurius",
+		npmPackage: "mercurius",
+		href: "https://mercurius.dev/",
+		description: "Mercurius is a GraphQL adapter for Fastify",
+		image: "https://github.com/mercurius-js.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Apollo Server",
+		author: "Apollo GraphQL",
+		gitHubRepo: "https://github.com/apollographql/apollo-server",
+		npmPackage: "@apollo/server",
+		href: "https://www.apollographql.com/docs/apollo-server/",
+		description:
+			"Apollo Server is an open-source, spec-compliant GraphQL server that's compatible with any GraphQL client, including Apollo Client. It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.",
+		image: "https://github.com/apollographql.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Helix",
+		author: "Contra",
+		gitHubRepo: "https://github.com/contra/graphql-helix",
+		npmPackage: "graphql-helix",
+		href: "https://www.graphql-helix.com/",
+		description:
+			"Build your own extensible and framework-agnostic GraphQL Server",
+		image:
+			"https://raw.githubusercontent.com/contra/graphql-helix/main/logo.svg",
+		tags: ["Servers"],
+	},
+	{
+		name: "Express-GraphQL",
+		author: "Meta",
+		gitHubRepo: "https://github.com/graphql/express-graphql",
+		npmPackage: "express-graphql",
+		href: "https://github.com/graphql/express-graphql",
+		description:
+			"Create a GraphQL HTTP server with any HTTP web framework that supports connect styled middleware, including Connect itself, Express and Restify.",
+		image: "https://github.com/graphql.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL-Go",
+		author: "graph-gophers",
+		gitHubRepo: "https://github.com/graph-gophers/graphql-go",
+		npmPackage: "",
+		href: "https://github.com/graph-gophers/graphql-go",
+		description:
+			"The goal of this project is to provide full support of the GraphQL draft specification with a set of idiomatic, easy to use Go packages.",
+		image: "https://github.com/graph-gophers.png",
+		tags: ["Servers"],
+	},
+	{
+		name: "Yoga",
+		author: "Dotan Simha",
+		gitHubRepo: "https://github.com/dotansimha/graphql-yoga",
+		npmPackage: "graphql-yoga@three",
+		href: "https://www.the-guild.dev/graphql/yoga-server",
+		description:
+			"Fully-featured GraphQL Server with focus on easy setup, performance & great developer experience",
+		image: "",
+		tags: ["Servers"],
+	},
+	{
+		name: "GraphQL .NET",
+		author: "",
+		gitHubRepo: "https://github.com/graphql-dotnet/graphql-dotnet",
+		npmPackage: "",
+		href: "https://graphql-dotnet.github.io/",
```

Comment:
I recommend swaping this to `https://graphql-dotnet.github.io/docs/getting-started/introduction`.

Old href:
![image](https://user-images.githubusercontent.com/20880360/200683478-89e981f6-99e7-456f-9650-0799d51952b7.png)

New href:
![Screen Shot 2022-11-08 at 1 51 10 PM](https://user-images.githubusercontent.com/20880360/200683460-f34a35f4-0b1b-48ed-9db0-9de361b03331.png)